pepper-apis/src/main/java/org/broadinstitute/ddp/constants/SqlConstants.java

```diff
@@ -73,6 +73,7 @@
         public static final String INSTANTIATE_UPON_REGISTRATION = "instantiate_upon_registration";
         public static final String MAX_INSTANCES_PER_USER = "max_instances_per_user";
         public static final String NAME_TRANS = "activity_name_trans";
+        public static final String SECOND_NAME_TRANS = "activity_second_name_trans";
```

Comment:
Anyone got a better name than "second name"?

Reply:
`name_pattern`? It looks like it's being used to provide an auto-naming scheme when there's multiple activities with the same code.

As implemented, this has potential i18n issues (you cannot always meaningfully add the number as a suffix) so I would recommend either adding printf-style substitution ("use %d to mark where the activity number should be added"), or a named parameter (a string search & replace with something like `{{count}}` could work). It's a common pattern in iOS development to use format strings in the translation files.

Reply:
Substitution is how it is working. If there is a "second name" available, we don't automatically append a number to the end. Instead, we render the text so you can put `$ddp.activityInstanceNumber()` anywhere  inside that text. I'm keeping the old behavior of "automatically append number" since that's how existing studies work -- if we want to get rid of  that, we need to go back and add/update activity definitions.

Reply:
Maybe something like follow up name?  Not sure that's really any better.

Reply:
Or SECOND_INSTANCE_NAME_TRANS.